src/engine/bank01.asm

```diff
@@ -5703,7 +5703,7 @@ Func_6435:
 	ld hl, wDuelTempList + 1
 	add hl, de
 	ld a, [hld]
-	cp $04
+	cp POKEMON_POWER
```

Comment:
This value is being compared to an element in wDuelTempList. Normally these are deck indices, but in this case I doubt it. In any case, POKEMON_POWER is probably what we don't want here (its okay to keep hexadecimal values when it's still not clear what it is)

Reply:
Changed it back to hex byte $04